###### Is gender independent of education level? A random sample of 395 people were surveyed and each person was asked to report the highest education level they obtained. The data that resulted from the survey is summarized in the following table:

###### Chi-Square Test Statistic
χ2=∑(O−E)2/E

where O represents the observed frequency. E is the expected frequency under the null hypothesis and computed by:
 
E = row total×column total / sample size

###### 	High School	 Bachelors	Masters	Ph.d.	Total

Female	60	54	46	41	201

Male	40	44	53	57	194

Total	100	98	99	98	395

###### Here's the table of expected counts:

 	High School	 Bachelors	Masters	Ph.d.	Total
    
Female	50.886	49.868	50.377	49.868	201

Male	49.114	48.132	48.623	48.132	194

Total	100	98	99	98	395

In [3]:
from scipy.stats import chisquare

In [4]:
chisquare([60,54,46,41,40,44,53,57], f_exp=[50.886, 49.868,50.377, 49.868, 49.114, 48.132, 48.623, 48.132])

Power_divergenceResult(statistic=8.0059071211091002, pvalue=0.33207330622863096)

###### The critical value of χ2 with 3 degree of freedom is 7.815. Since 8.006 > 7.815, therefore we reject the null hypothesis and conclude that the education level depends on gender at a 5% level of significance.